### ISSUE TITLE
Normalization of data

### ISSUE THREAD

**Issue by mattiaswiklund99** [neuraloperator/neuraloperator#134]
Hi! I looked over the fourier_3d script and I'm wondering about the decoding and encoding of the data. It seems that you first normalize (encode function) the data and then denormalize (decode function) it before traning. From what I've read, one usually normalizes the data before training. What is the reason for the denormalization?

**Comment by JeanKossaifi** [comment]
You may want to switch to the latest version on main. You train on encoded inputs, but for testing to get relative error you may want to decode the predicted outputs back to the original space, or directly predict unnormalized outputs.

**Comment by kallegrens** [comment]
What @mattiaswiklund99 is saying is that in the version used get the results of the paper, you actually train on _decoded_ data, which is a bit weird.

**Comment by zongyi-li** [comment]
The normalized loss and original loss (decoded) are different. It's optional to train on either the normalized loss or the original loss. The final goal is to reduce the error on the original scale. Sometimes training on the normalized loss will cause overfitting, so we choose to directly train on the original loss.
